lua/methods/atmosphere.lua

```diff
@@ -0,0 +1,56 @@
+--
+-- RPC atmosphere actions
+-- https://wiki.hoggitworld.com/view/DCS_singleton_atmosphere
+--
+
+local atmosphere = atmosphere
+local coord = coord
+local GRPC = GRPC
+
+local function convertWindResults(windvec3)
+  local direction = math.deg(math.atan2(windvec3.z, windvec3.x))
+  
+  if direction < 0 then
+    direction = direction + 360
+  end
+  
+  -- Convert TO direction to FROM direction. 
+  if direction > 180 then
+    direction = direction-180
+  else
+    direction = direction+180
```

Comment:
This will convert `180` to `360` and not `0`, which is fine for me, just want to mention it in case you were aiming at a `000` rather than `360`

Reply:
Good catch. This is intentional.